### PR TITLE
Remove PowerMock dependencies.

### PR DESCRIPTION
**Pull request by madurangasiriwardena**

### CODE REVIEW

File: components/org.wso2.carbon.identity.organization.management.authn/src/test/java/org/wso2/carbon/identity/organization/management/authn/OrganizationAuthenticatorTest.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
  *
  * WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (wenodpathirana) [inline components/org.wso2.carbon.identity.organization.management.authn/src/test/java/org/wso2/carbon/identity/organization/management/authn/OrganizationAuthenticatorTest.java:4]:
```suggestion
 * WSO2 LLC. licenses this file to you under the Apache License,
```

File: components/org.wso2.carbon.identity.organization.management.authn/src/test/java/org/wso2/carbon/identity/organization/management/authn/OrganizationAuthenticatorTest.java [hunk 2]
```diff
@@ -91,98 +88,63 @@ public class OrganizationAuthenticatorTest extends PowerMockTestCase {
...
+        mockAuthenticationContext = mock(AuthenticationContext.class);
+        mockRealmService = mock(RealmService.class);
+        mockOrganizationManager = mock(OrganizationManager.class);
+        mockOrgApplicationManager = mock(OrgApplicationManager.class);
+        mockServiceProvider = mock(ServiceProvider.class);
+        mockInboundAuthenticationConfig = mock(InboundAuthenticationConfig.class);
+        mockOAuthAdminServiceImpl = mock(OAuthAdminServiceImpl.class);
+        mockOAuthConsumerAppDTO = mock(OAuthConsumerAppDTO.class);
+        mockExternalIdPConfig = mock(ExternalIdPConfig.class);
+        mockOrganization = mock(Organization.class);
```
Comment (dewniMW) [inline components/org.wso2.carbon.identity.organization.management.authn/src/test/java/org/wso2/carbon/identity/organization/management/authn/OrganizationAuthenticatorTest.java:120]: shall we have this with @BeforeClass ?

File: pom.xml [hunk 3]
```diff
@@ -554,9 +565,11 @@
         <findsecbugs-plugin.version>1.10.1</findsecbugs-plugin.version>
         <findbugs.annotations.version>3.0.1</findbugs.annotations.version>
         <carbon.p2.plugin.version>5.1.2</carbon.p2.plugin.version>
+        <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
 
         <compiler-source.version>1.8</compiler-source.version>
         <compiler-target.version>1.8</compiler-target.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <mockito-core.version>4.6.1</mockito-core.version>
```
Comment (dewniMW) [inline pom.xml:573]: shall we move this to https://github.com/wso2-extensions/identity-organization-management/pull/112/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R558?

Comment (dewniMW) [reply pom.xml:573]: Also let's remove https://github.com/wso2-extensions/identity-organization-management/pull/112/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R553

File: components/org.wso2.carbon.identity.organization.management.authn/src/test/java/org/wso2/carbon/identity/organization/management/authn/OrganizationAuthenticatorTest.java (hunk 2 above)
Comment (madurangasiriwardena) [reply components/org.wso2.carbon.identity.organization.management.authn/src/test/java/org/wso2/carbon/identity/organization/management/authn/OrganizationAuthenticatorTest.java:120]: Due to some reason this is not working. Will need some time to figure this out. In the meantime, we can proceed with this PR.

**Review by dewniMW — APPROVED** [review]

**Pull request merged**